### DISCUSSION TITLE
If the FreeRDP dll generated with Media Foundation option then freerdp is not working in all the machine

### DISCUSSION THREAD

**Discussion by SolairajMariappan** [FreeRDP/FreeRDP#6736] [Q&A]
If the FreeRDP dll generated with Media Foundation option then freerdp is not working in all the machine. Its is working only when the machine is enabled with the feature. Is there work around to have the dll which is supported commonly with out any prerequisites ?

**Comment by akallabeth** [comment]
Use a build with `openh264` or `ffmpeg` as `h264` backend.
`MediaFoundation` backend is failing whenever the desired format is not supported.
